src/test/java/com/coutemeier/maven/plugins/asciidoctor/lifecycle/SimpleDavServer.java

```diff
@@ -0,0 +1,52 @@
+package com.coutemeier.maven.plugins.asciidoctor.lifecycle;
+
+import com.coutemeier.maven.plugins.asciidoctor.lifecycle.util.FileUtil;
+
+import de.bitinsomnia.webdav.server.MiltonWebDAVFileServer;
+
+import java.io.File;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SimpleDavServer {
+    private Logger logger = LoggerFactory.getLogger( getClass() );
+
+    private final File rootFolder;
+
+    private final int port;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'port' could be replaced by a local variable.](https://app.codacy.com/manual/asciidoctor-lifecycle-maven-plugin/asciidoctor-lifecycle-maven-plugin/pullRequest?prid=5403658)